src/mainboard/clevo/adl-p/vboot-rwa.fmd

```diff
@@ -0,0 +1,40 @@
+FLASH@0xff000000 0x1000000 {
+	SI_ALL 5M {
```

Comment:
Incorrect on this platform. `util/ifdtool` tells me that the the FD + ME portion spans from `0x0` to `0x4b6fff` (seems like Intel doesn't round these to MiB anymore)